tsconfig.json

```diff
@@ -56,13 +56,7 @@
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
   },
   "include": [
-    "types/Nimiq.d.ts",
-    "types/Keyguard.d.ts",
-    "types/ErrorContainer.d.ts",
-    "types/MessagePrefix.d.ts",
-    "types/BarcodeScanner.d.ts",
-    "types/BitcoinJS.d.ts",
-    "types/UserAgentClientHints.d.ts",
+    "types/*",
```

Comment:
Note that this now also includes `KeyguardRequestNamespace.d.ts`.
Don't know whether that's any issue though.

Reply:
It's also included via `Keyguard.d.ts`, but the CI still works, so it doesn't seem to be a problem that it's doubled.

Reply:
Actually removed the type reference in `Keyguard.d.ts` now, so it's no longer double.